pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -967,8 +967,8 @@ self: super: builtins.intersectAttrs super {
         wrapProgram "$out/bin/nvfetcher" --prefix 'PATH' ':' "${
           pkgs.lib.makeBinPath [
             pkgs.nvchecker
-            pkgs.nix-prefetch
-            pkgs.nix-prefetch-docker
```

Comment:
I believe `nix-prefetch-docker` is still required.

Reply:
Fixed